packages/location-management/src/components/LocationTagAddEdit/Form.tsx

```diff
@@ -0,0 +1,163 @@
+import * as Yup from 'yup';
+import React, { useEffect, useState } from 'react';
+import { SubmitButton, Form as AntForm, Input, Radio } from 'formik-antd';
+import { notification, Button } from 'antd';
+import { history } from '@onaio/connected-reducer-registry';
+import { OpenSRPService } from '@opensrp/server-service';
+import { getAccessToken } from '@onaio/session-reducer';
+import { Formik } from 'formik';
+import { useSelector } from 'react-redux';
+import { API_BASE_URL, LOCATION_TAG_ALL, LOCATION_TAG_GET } from '../../constants';
+import {
+  LocationTag,
+  LocationTagPayloadPOST,
+  LocationTagPayloadPUT,
+} from '../../ducks/location-tags';
+import { Ripple } from '@onaio/loaders';
+
+const layout = { labelCol: { span: 8 }, wrapperCol: { span: 11 } };
+const offsetLayout = { wrapperCol: { offset: 8, span: 11 } };
+
+const status = [
+  { label: 'Active', value: true },
+  { label: 'Inactive', value: false },
+];
+
+interface FormField {
+  name: string;
+  description: string;
+  active: boolean;
+}
+
+/** yup validations for practitioner data object from form */
+const userSchema = Yup.object().shape({
+  name: Yup.string().typeError('Name must be a String').required('Name is Required'),
+  active: Yup.boolean().typeError('Status must be a Boolean').required('Status is Required'),
+  description: Yup.string()
+    .typeError('Description must be a String')
+    .required('Description is Required'),
+});
+
+interface Props {
+  id?: string;
+}
+
+/**
+ * Handle form submission
+ *
+ * @param {Object} values the form fields
+ * @param {string} accessToken api access token
+ * @param {object} props component props
+ * @param {Function} setSubmitting method to set submission status
+ */
+export const onSubmit = (
+  values: FormField,
+  accessToken: string,
+  props: Props,
+  setSubmitting: (isSubmitting: boolean) => void
+) => {
+  const serve = new OpenSRPService(accessToken, API_BASE_URL, LOCATION_TAG_ALL);
+
+  const payload: LocationTagPayloadPOST | LocationTagPayloadPUT = values;
+
+  if (props.id) {
+    (payload as LocationTagPayloadPUT).id = props.id;
+    serve
+      .update(payload)
+      .then(() => {
+        notification.success({ message: 'Location Tag updated successfully', description: '' });
+        setSubmitting(false);
+        history.goBack();
+      })
+      .catch((e: Error) => {
+        notification.error({ message: `${e}`, description: '' });
+        setSubmitting(false);
+      });
+  } else {
+    serve
+      .create(payload)
+      .then(() => {
+        notification.success({ message: 'Location Tag successfully', description: '' });
+        setSubmitting(false);
+        history.goBack();
+      })
+      .catch((e: Error) => {
+        notification.error({ message: `${e}`, description: '' });
```

Comment:
Let's use the package @opensrp/notifications package for notifications. Also, let' show the user the generic An error occurred

Reply:
done replacing with @opensrp/notifications...  

should we log the error into console? cuz without that it will be harder to debug the problem later on in future.

Reply:
done